google/cloud/bigtable/internal/mutate_rows_limiter.h

```diff
@@ -31,20 +31,25 @@ GOOGLE_CLOUD_CPP_INLINE_NAMESPACE_BEGIN
 /// A Bigtable-specific wrapper over the more generic `RateLimiter`
 class MutateRowsLimiter {
  public:
+  virtual ~MutateRowsLimiter() = 0;
```

Comment:
Q.  What do we think we're buying here?

Reply:
Whatever we are buying in the stubs? (I am not someone who understands things. I merely recognize patterns.)

I know that if I do not default this virtual destructor the code stops compiling.

Reply:
My question is why make it pure (causing you to then need to explicitly define it, here and in subclasses).  The class already has other pure functions, so either way it is not instantiatable.  That is, if you want to provide for deletion through the base class, why isn't just `virtual ~MutateRowsLimiter() = default` sufficient?

Reply:
Done.

Reply:
It seems, now, that you don't need the new, explicit definitions of the dtors in the subclasses.  (You may also want to update the PR description.)

Reply:
Done and done.